src/components/Login.jsx

```diff
@@ -8,6 +8,15 @@ import { styles } from '../styling/styles';
 import { createSession } from '../controllers/sessionController';
 import { LoginValidationSchema } from '../utils/formValidationSchemas';
 
+/**
+ * Render a form for user login, validate using a yup schema.
+ * On submission, send login data to the server.
+ *
+ * @param {Function} handleSubmit - handle form submission;
+ *  args: form values
+ * (username, password)
+ * @returns {React.JSX.Element}
+ */
 const LoginForm = ({ updateUser }) => {
```

Comment:
I think this function does not actually get `handleSubmit` function as a parameter. As far as I can tell this function has one parameter, which is an object with `updateUser` key having a function as value. Then the JSDoc would become something like

```
@param {{ updateUser: Function }} props 
```